diff -up gutenprint-5.2.5/src/gutenprintui2/Makefile.am.yyin gutenprint-5.2.5/src/gutenprintui2/Makefile.am
--- gutenprint-5.2.5/src/gutenprintui2/Makefile.am.yyin	2008-10-24 02:41:04.000000000 +0200
+++ gutenprint-5.2.5/src/gutenprintui2/Makefile.am	2016-10-03 15:50:38.318761765 +0200
@@ -55,7 +55,8 @@ libgutenprintui2_la_CFLAGS = $(AM_CFLAGS
 # Uncomment to build a versioned library
 libgutenprintui2_la_LDFLAGS = \
  -version-info  $(GUTENPRINTUI2_CURRENT_INTERFACE):$(GUTENPRINTUI2_INTERFACE_AGE):$(GUTENPRINTUI2_BINARY_AGE) \
- -rpath $(libdir) -no-undefined
+ -rpath $(libdir) -no-undefined \
+ -export-symbols-regex '^([^y]|y[^y])'
 
 AM_LFLAGS = -i
 AM_YFLAGS = -d
diff -up gutenprint-5.2.5/src/gutenprintui2/Makefile.in.yyin gutenprint-5.2.5/src/gutenprintui2/Makefile.in
--- gutenprint-5.2.5/src/gutenprintui2/Makefile.in.yyin	2016-10-03 18:53:51.812070939 +0200
+++ gutenprint-5.2.5/src/gutenprintui2/Makefile.in	2016-10-03 18:54:58.593577928 +0200
@@ -344,7 +344,8 @@ libgutenprintui2_la_CFLAGS = $(AM_CFLAGS
 # Uncomment to build a versioned library
 libgutenprintui2_la_LDFLAGS = \
  -version-info  $(GUTENPRINTUI2_CURRENT_INTERFACE):$(GUTENPRINTUI2_INTERFACE_AGE):$(GUTENPRINTUI2_BINARY_AGE) \
- -rpath $(libdir) -no-undefined
+ -rpath $(libdir) -no-undefined \
+ -export-symbols-regex '^([^y]|y[^y])'
 
 AM_LFLAGS = -i
 AM_YFLAGS = -d
